src/lib/spinel/spinel.h

```diff
@@ -78,20 +78,20 @@
  *
  *   The Interface Identifier (IID) is a number between 0 and 3, which
  *   is associated by the OS with a specific NCP. This allows the protocol
- *   to support up to 4 NCPs under same connection.
+ *   to support multiple networks under same connection.
  *
  *   The least significant bits of the header represent the Transaction
  *   Identifier (TID). The TID is used for correlating responses to the
  *   commands which generated them.
  *
  *   When a command is sent from the host, any reply to that command sent
- *   by the NCP will use the same value for the TID.  When the host
- *   receives a frame that matches the TID of the command it sent, it can
- *   easily recognize that frame as the actual response to that command.
+ *   by the NCP will use the same value for the IID and TID.  When the host
+ *   receives a frame that matches the IID and TID of the command it sent, it
+ *   can easily recognize that frame as the actual response to that command.
  *
- *   The TID value of zero (0) is used for commands to which a correlated
- *   response is not expected or needed, such as for unsolicited update
- *   commands sent to the host from the NCP.
+ *   The IID and TID value of zero (0) is used for commands to which a
```

Comment:
Is the addition of `IID` here correct? I am not sure.
`TID = 0` indicates no response is needed. We can use `IID` zero with a non-zero TID and we expect a response.

Reply:
You are correct. I updated the text now. Please let me know how it looks.